scripts/builtin/img_mirror_linearized.dml

```diff
@@ -0,0 +1,83 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+# This function has  the same functionality with img_mirror but it handles multiple images at
+# the same time. Each row of the input and output matrix represents a linearized image/matrix
+# It flips an image on the X (horizontal) or Y (vertical) axis.
+# INPUT:
+# -----------------------------------------------------------------------------------------
+# img_matrix           Input matrix/image (every row represents a linearized matrix/image)
+# horizontal_axis      flip either in X or Y axis
+# original_rows        number of rows in the original 2-D images 
+# original_cols        number of cols in the original 2-D images 
+# -----------------------------------------------------------------------------------------
+#
+# OUTPUT:
+# ----------------------------------------------------------------------------------------------------------------------
+# img_out  Output matrix/image  (every row represents a linearized matrix/image)
+# ----------------------------------------------------------------------------------------------------------------------
+
+m_img_mirror_linearized = function(matrix[double] img_matrix, Boolean horizontal_axis,Integer original_rows ,Integer original_cols) return (matrix[double] R){
+  n =  ncol(img_matrix);
+  R = matrix(0, rows=nrow(img_matrix), cols=n); #initialize the output_matrix
+  rows =original_rows
+  cols= original_cols
+
+  
+  if (horizontal_axis) {
+
+   for (i in seq(1,(rows%/% 2)*cols,cols)){
```

Comment:
one indentation layer is two spaces here you used one.

Reply:
corrected the indention layers